File: 240214/함수를 이용한 369 게임/369-games-using-functions.py
a, b = map(int, input().split())

def triple(n):
    return n % 3 == 0

def in_369(n):
    return '3' in str(n) or '6' in str(n) or '9' in str(n)

cnt = 0

for i in range(a, b+1):
    if triple(i) or in_369(i):
        cnt += 1

print(cnt)

File: 240805/함수를 이용한 369 게임/369-games-using-functions.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ").map(Number);
let a = input[0];
let b = input[1];
let cnt = 0;

function check369(n) {
    let arr = String(n).split("").map(Number);
    return arr.includes(3) || arr.includes(6) || arr.includes(9)
}

function isMagicNumber(n) {
    return n % 3 === 0 || check369(n);
}

for (let i = a; i <= b; i++) {
    if (isMagicNumber(i)) {
        cnt++;
    }
}

console.log(cnt);